docs/api/process.md

```diff
@@ -171,6 +171,13 @@ Returns `Object`:
 Returns an object giving memory usage statistics about the entire system. Note
 that all statistics are reported in Kilobytes.
 
+### `process.takeHeapSnapshot(filePath)`
+
+* `filePath` String - Path to the output file.
+
+Takes a V8 heap snapshot and saves it to `filePath`.
+Throws an exception in case of failure.
```

Comment:
We shouldn't make a worse API because the docs generator is breaking the PR.  This should be returning a success state not throwing an error.  Working around a docs generator bug by changing the API is a bad idea.

Requesting changes and as mentioned I'll look into the docs generator issue

Reply:
waiting for https://github.com/electron/electron-typescript-definitions/releases/tag/v2.0.1 to be published to NPM

Reply:
@miniak It was published when that github release went out an hour or so ago --> https://www.npmjs.com/package/electron-typescript-definitions/v/2.0.1

It's an automated pipeline

Reply:
The npmjs webpage took a while to reflect the update.

Reply:
> Working around a docs generator bug by changing the API is a bad idea.

I agree that generally it is a bad idea.

> This should be returning a success state not throwing an error.

But in this case I would argue.
In most cases an exception will be thrown because of a user "error", like an invalid file path or lack of permission to write to a specified location.
Returning just a boolean "state" value simply doesn't provide any useful feedback to a user.

Reply:
@alexeykuzmin Sure if we have things like `EACCESS`, `EPERM` and other such standard file errors then throwing them to be consistent with things like node makes sense.  From what I saw this was just throwing a generic "something went wrong" error which is just as helpful as returning false but harder / more annoying to catch.